src/main/java/com/esri/samples/scene/change_atmosphere_effect/ChangeAtmosphereEffectSample.java

```diff
@@ -0,0 +1,142 @@
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AtmosphereEffect;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.mapping.view.Camera;
+
+public class ChangeAtmosphereEffectSample extends Application {
+
+  private SceneView sceneView;
+  private static final String ELEVATION_IMAGE_SERVICE =
+      "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and JavaFX app scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Change Atmosphere Effect Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a control panel
+      VBox controlsVBox = new VBox(6);
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0, 0, 0, 0.3)"), CornerRadii.EMPTY, Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setMaxSize(260, 110);
+      controlsVBox.getStyleClass().add("panel-region");
+
+      // create buttons for interaction
+      Button noAtmosphereButton = new Button("No Atmosphere Effect");
+      Button realisticAtmosphereButton = new Button ("Realistic Atmosphere Effect");
+      Button horizonAtmosphereButton = new Button ("Horizon Only Atmosphere Effect");
+      noAtmosphereButton.setMaxWidth(Double.MAX_VALUE);
+      realisticAtmosphereButton.setMaxWidth(Double.MAX_VALUE);
+      horizonAtmosphereButton.setMaxWidth(Double.MAX_VALUE);
+
+      noAtmosphereButton.setOnAction(event -> {
+        // set atmosphere effect to none
+        sceneView.setAtmosphereEffect(AtmosphereEffect.NONE);
+      });
+
+      realisticAtmosphereButton.setOnAction(event -> {
+        // set atmosphere effect to realistic
+        sceneView.setAtmosphereEffect(AtmosphereEffect.REALISTIC);
+      });
+
+      horizonAtmosphereButton.setOnAction(event -> {
+        // set atmosphere effect to horizon only
+        sceneView.setAtmosphereEffect(AtmosphereEffect.HORIZON_ONLY);
+      });
+
+      // add controls to the user interface pane
+      controlsVBox.getChildren().addAll(noAtmosphereButton, realisticAtmosphereButton, horizonAtmosphereButton);
+
+      // create a scene and add a basemap to it
```

Comment:
I would move the scene, scene view, surface, and camera code above the control box.